app/assets/stylesheets/_variables.scss

```
@@ -1,3 +1,17 @@
 $primary-light: #ffa9a9;
 $primary-dark: #a20202;
 $primary: #d01659;
+
+// Whites
+$white-100: #eef1ff;
+$white-500: #fff;
+
+// Blacks
+$black-100: #3d3c42;
+$black-500: #000;
+
+// Colors
+$red: #f00;
+
+// Variables
+$_auth-form-pads: 40px 50px 20px;
```

Could we use the rem here?

```suggestion
$_auth-form-pads: 2.5rem 3.125rem 1.25rem; // 40px 50px 20px
```